### PR TITLE
SASS-918 - Added DeleteEmploymentConnector

### PR DESCRIPTION
**Pull request by Dula-J**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
https://jira.tools.tax.service.gov.uk/browse/SASS-918

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you addressed warnings where appropriate?

##### After PRs been Reviewed
- [ ]  Have you checked the PR Builder passes?
- [ ]  Have you checked for merge conflicts?
- [ ]  Have you checked code coverage isn’t lower than previously?

### CODE REVIEW

File: test/services/EmploymentServiceSpec.scala [hunk 1]
```diff
@@ -25,22 +25,25 @@ import utils.TestUtils
 
 import scala.concurrent.Future
 
-class CreateEmploymentServiceSpec extends TestUtils {
+class EmploymentServiceSpec extends TestUtils {
 
-  val mockConnector = mock[CreateEmploymentConnector]
-  val createEmploymentService = new CreateEmploymentService(mockConnector)
+  val mockCreateEmploymentConnector = mock[CreateEmploymentConnector]
```
Comment (joeyymoore) [inline test/services/EmploymentServiceSpec.scala:30]: These two vals for the connectors give warnings about needing type annotation, if you wanna take care of them?

File: test/services/EmploymentServiceSpec.scala [hunk 2]
```diff
@@ -65,4 +68,34 @@ class CreateEmploymentServiceSpec extends TestUtils {
...
+
+        val result = createEmploymentService.deleteEmployment(nino, taxYear, employmentId)
+
+        await(result) mustBe Right(())
+      }
+    }
+
+    "return Left containing DesError" when {
+      "the deleteEmployment connector call fails" in {
+        val desError = DesErrorModel(500, DesErrorBodyModel("DES_CODE", "DES_REASON"))
```
Comment (joeyymoore) [inline test/services/EmploymentServiceSpec.scala:88]: This 500 and the one on line 58 give magic number weak warnings, could be turned into INTERNAL_SERVER_ERROR's if you want to get rid of them?

**Comment by joeyymoore** [comment]
You need to rebase with master

File: test/services/EmploymentServiceSpec.scala (hunk 1 above)
Comment (Dula-J) [reply test/services/EmploymentServiceSpec.scala:30]: will do

File: test/services/EmploymentServiceSpec.scala (hunk 2 above)
Comment (Dula-J) [reply test/services/EmploymentServiceSpec.scala:88]: yep

**Pull request merged**